src/sadhana/SecondMaximum.java

```diff
@@ -0,0 +1,36 @@
+package sadhana;
+
+/*Assignment - 29 : 26th Nov'2020 
+Find second maximum from array without sorting the array. */
+
+public class SecondMaximum {
+	void printData(int arr[]) {
+		int largest = arr[0];
+		int secondLargest = arr[0];
+		System.out.println("The given array is:");
+		for (int i = 0; i < arr.length; i++) {
+			System.out.print(arr[i] + ",");
+		}
+		for (int i = 0; i < arr.length; i++) {
+
+			if (arr[i] > largest) {
+				secondLargest = largest;
+				largest = arr[i];
+
+			} else if (arr[i] > secondLargest) {
```

Comment:
Here you should check below on more condition as well

else if (arr[i] > secondLargest && arr[i] != largest) {
}